src/xdocs/checks/misc/uncommentedmain.xml

```diff
@@ -58,11 +58,11 @@
         <p id="Example1-code">Example:</p>
         <source>
 public class Game {
-   public static void main(String... args){}   // violation
+   public static void main(String... args){}   // violation 'Uncommented main method found'
```

Comment:
do not directly modify .xml file, first modify it's .xml.template file, in this case it is: https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/checks/misc/uncommentedmain.xml.template

then run `mvn clean verify`, it will generate content in your .xml file ( in this case uncommentedmain.xml )

otherwise your build will keep failing

Reply:
Thanks for the help @Zopsss . Doing it.